egs/aishell/s10/chain/inference.py

```diff
@@ -28,7 +29,11 @@ def main():
         logging.warning('No GPU detected! Use CPU for inference.')
         device = torch.device('cpu')
     else:
-        device = torch.device('cuda', args.device_id)
+        devices = allocate_gpu_devices(1)
```

Comment:
What if the user provides `device_id` at the commandline?

Reply:
Let me do this in another pr, I think what we need is `device_ids` instead of `device_id`. Actually I don't think about this very clearly now as: 

- Multiple gpus is required when training with multiple GPUs on a single machine, that's why we need `device_ids`

- Assign GPU id when training with multiple machines is relatively complex for uses.

@danpovey any comments about this?